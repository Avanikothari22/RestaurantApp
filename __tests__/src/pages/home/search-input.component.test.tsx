import SearchInput from '../../../../src/pages/home/search-input.component';
import Home from '../../../../src/pages/home/home';
import React from 'react';
import renderer from 'react-test-renderer';
// beforeAll(() => { 
//     jest.mock('@react-native-community/async-storage');
//   })
  test('renders correctly', () => {
    const component = renderer.create(
        <SearchInput/>
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
  });