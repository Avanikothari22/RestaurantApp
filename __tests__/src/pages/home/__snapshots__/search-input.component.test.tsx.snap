// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container"
>
  <div
    style={
      Object {
        "backgroundImage": "url([object Object])",
        "backgroundPosition": "center center",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "cover",
        "height": 307.2,
      }
    }
  >
    <h2
      className="Search-Heading"
    >
      Search Near By Restaurants
    </h2>
    <input
      className="Search-Input"
      onChange={[Function]}
      placeholder="Search by cuisine or restaurant"
      type="text"
      value=""
    />
    <input
      className="Search-Button"
      onClick={[Function]}
      style={
        Object {
          "borderRadius": 5,
        }
      }
      type="button"
      value="Search"
    />
  </div>
  <div
    className="MainContainer"
  >
    <div
      className="sweet-loading"
    />
    
  </div>
</div>
`;
